catalog/plugin/catalog-plugin-federation-replication/src/test/java/ddf/catalog/federation/layered/TestPlugin.java

```diff
@@ -64,13 +58,9 @@
 
 public class TestPlugin {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(TestPlugin.class);
-
     // I changed the port so that it would not conflict in testing with other services
     private static final String ENDPOINT_ADDRESS = "http://localhost:8282/services/catalog";
 
-    private static final String WADL_ADDRESS = ENDPOINT_ADDRESS + "?_wadl";
-
     private static MockRestEndpoint endpoint;
 
     private static Server server;
```

Comment:
Alright, do you mean Before/AfterClass methods in this file specifically, or throughout the whole project?